src/views/Layout.jsx

```diff
@@ -1,4 +1,4 @@
-import { Outlet } from 'react-router-dom';
+import { Outlet, NavLink } from 'react-router-dom';
 
 import './Layout.css';
```

Comment:
Looks beautiful!!! 👏 👏 👏 

@DevinaG007 @andiedoescode Now that you've implemented these changes, you can also go ahead and delete the TODO comment here in Layout.jsx from lines 7-13 to keep things nice and tidy! ✅